src/admin-integration-email-add/admin-integration-email-add.html

```diff
@@ -0,0 +1,19 @@
+<div class="modal" role="dialog" aria-hidden="true">
+  <div class="modal-dialog">
+      <div class="modal-content">
+          <div class="modal-header">
+              <h1>{{vm.modalHeaderKey | message}}</h1>
```

Comment:
please add one-time bindings, for example `{{:: vm.modalHeaderKey | message}}` for all texts, if the value will not be changed

https://docs.angularjs.org/guide/expression#one-time-binding